### ISSUE TITLE
PHPDoc order, the Symfony way

### ISSUE THREAD

**Issue by Soullivaneuh** [FriendsOfPHP/PHP-CS-Fixer#1983]
Currently, the `phpdoc_order` sorts the annotations like this:

``` php
/**
 * Transforms the input given as first argument.
 *
 * @param bool|string $dummy   Some argument description
 * @param array       $options An options collection to be used within the transformation
 *
 * @throws \RuntimeException When an invalid option is provided
 *
 * @return string|null The transformed input
 */
private function transformText($dummy, array $options = array())
```

But, according to the [Symfony Coding Standard sample](http://symfony.com/doc/current/contributing/code/standards.html), this should be like this:

``` php
/**
 * Transforms the input given as first argument.
 *
 * @param bool|string $dummy   Some argument description
 * @param array       $options An options collection to be used within the transformation
 *
 * @return string|null The transformed input
 *
 * @throws \RuntimeException When an invalid option is provided
 */
private function transformText($dummy, array $options = array())
```

In a nutshell, `@throws` should be after `@return`, not before.

It would be great to have another fixer to be integrated directly to the Symfony group.

What do you think?

cc @GrahamCampbell (seems you made this fixer)

**Comment by SpacePossum** [comment]
:+1:

**Comment by keradus** [comment]
:+1:
I already requested somewhere that this fixer should be configurable

**Comment by Soullivaneuh** [comment]
> I already requested somewhere that this fixer should be configurable

If we make it configurable, how to integrate the Symfony configuration automatically on Symfony preset?

**Comment by keradus** [comment]
By adjusting `@Symfony` ruleset

**Comment by Soullivaneuh** [comment]
@keradus So should this be done directly on master for the option?

**Comment by SpacePossum** [comment]
There is no need, a new feature should be done on 1.12 and will be merged upstream.
The exception are BC breaking features or features that are very hard to do on 1.12 (for example if you rely on transformers or other features not available on 1.12)

**Comment by keradus** [comment]
Closing as duplicate of #1602
